### PR TITLE
Remove generation of unneeded secret

### PR DESCRIPTION
**Pull request by wizard580**
#### What this PR does / why we need it:
If/when I define env var GF_SECURITY_ADMIN_PASSWORD this chart will make K8S secret with auto-generated admin password, but this password will not be used. 

This PR fixes this.

Please review if I missed anything.

Thanks.

### CODE REVIEW

**Comment by zanhsieh** [comment]
@wizard580 
Thanks for contribution.
Please sign DCO and bump the chart to latest version as seen in the bottom of this PR (click Details for instructions).

**Comment by thesuperzapper** [comment]
@wizard580, it's really not clear why `stable/airflow` commits have made their way into your PR, perhaps try rebase off remote master?

**Comment by thesuperzapper** [comment]
@zanhsieh, it's weirder than that, those commits are not from this PR, they are just recent commits to the `stable/airflow`

**Comment by zanhsieh** [comment]
@wizard580 @thesuperzapper 
Please rebase.

**Comment by wizard580** [comment]
Sorry for mistakes. Fixed.

**Review by zanhsieh — APPROVED** [review]

**Review by zanhsieh — APPROVED** [review]

File: stable/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.1.0
+version: 5.1.1
```
Comment (zanhsieh) [inline stable/grafana/Chart.yaml]: Could you bump to 5.1.2 please? Thanks you.

**Review by zanhsieh — CHANGES_REQUESTED** [review]

**Review by zanhsieh — APPROVED** [review]

**Comment by zanhsieh** [comment]
@wizard580 
Could you resolve stable/grafana/Chart.yaml conflict listed on the bottom of this PR?

**Pull request merged**